girder/web_client/src/rest.js

```diff
@@ -137,28 +137,9 @@ let restRequest = function (opts) {
         delete args.girderToken;
     }
 
-    let jqXHR = Backbone.$.ajax(args);
-    jqXHR.error = function () {
```

Comment:
@zachmullen Just to confirm, you grep-ed for usages of `.fail` on `restRequest` promises? In my experience, this has been one of the most stubborn to remove.

Reply:
I didn't, thanks for reminding me, I assumed whoever deprecated it had done so. I'll go through and fix them, unless you think this is too risky and we should just leave the fallback in.

Reply:
I was confused, it's actually `.error` that needs to be removed. I could not find any instances of that remaining.